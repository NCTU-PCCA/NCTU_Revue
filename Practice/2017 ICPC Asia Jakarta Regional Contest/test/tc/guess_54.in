58198162692282686 56
><<<<<>><>><<>><><><<><<<><<><><<<><><<<><>>>><<><<<>><>
