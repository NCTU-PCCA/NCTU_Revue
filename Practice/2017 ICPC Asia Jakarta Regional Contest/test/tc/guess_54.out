1 58198162692282686 58198162692282685 58198162692282684 58198162692282683 58198162692282682 2 3 58198162692282681 4 5 58198162692282680 58198162692282679 6 7 58198162692282678 8 58198162692282677 9 58198162692282676 58198162692282675 10 58198162692282674 58198162692282673 58198162692282672 11 58198162692282671 58198162692282670 12 58198162692282669 13 58198162692282668 58198162692282667 58198162692282666 14 58198162692282665 15 58198162692282664 58198162692282663 58198162692282662 16 58198162692282661 17 18 19 20 58198162692282660 58198162692282659 21 58198162692282658 58198162692282657 58198162692282656 22 23 58198162692282655 24
